scripts/clickhouse_import_support/copy_mysql_database_tables_to_clickhouse.sh

```diff
@@ -0,0 +1,263 @@
+#!/usr/bin/env bash
+
+# load dependencies
+unset this_script_dir
+this_script_dir="$(dirname "$(readlink -f $0)")"
+if ! source "$this_script_dir/parse_property_file_functions.sh" ; then
+    echo "Error : unable to load dependency : $this_script_dir/parse_property_file_functions.sh" >&2
+    exit 1
+fi
+if ! source "$this_script_dir/sling_command_line_functions.sh" ; then
+    echo "Error : unable to load dependency : $this_script_dir/sling_command_line_functions.sh" >&2
+    exit 1
+fi
+unset this_script_dir
+
+function usage() {
+    echo "usage:"
+}
+
+# other non-local environment variables in use
+unset my_properties
+unset database_table_list
+unset clickhouse_destination_database_name
+unset mysql_source_database_name
+declare -A my_properties
+declare -a database_table_list
+declare -A table_has_been_copied_and_verified
+declare -A mysql_database_table_record_count
+mysql_table_record_count=""
+clickhouse_destination_database_name=""
+mysql_source_database_name=""
+database_table_list_filepath="$(pwd)/cmd_database_table_list.txt"
+mysql_table_record_count_filepath="$(pwd)/cmd_mysql_table_record_count.txt"
+clickhouse_table_record_count_filepath="$(pwd)/cmd_clickhouse_table_record_count.txt"
+copy_table_contents_with_sling_filepath="$(pwd)/cmd_copy_table_contents_with_sling.txt"
+clickhouse_is_responsive_filepath="$(pwd)/cmd_clickhouse_is_responsive.txt"
+SECONDS_BETWEEN_RESPONSIVENESS_RETRY=$((60))
+SECONDS_BETWEEN_COPY_RETRY=$((15*60))
+
+function initialize_main() {
+    if ! [ "$database_to_transfer" == "blue" ] && ! [ "$database_to_transfer" == "green" ] ; then
+        echo "Error : argument for database_to_transfer must be either 'blue' or 'green'" >&2
+        return 1
+    fi
+    if ! parse_property_file "$properties_filepath" my_properties ; then
+        usage
+        return 1
+    fi
+    if [ "$database_to_transfer" == "blue" ] ; then
+        clickhouse_destination_database_name="${my_properties['clickhouse_blue_database_name']}"
+        mysql_source_database_name="${my_properties['mysql_blue_database_name']}"
+    else
```

Comment:
Here we could check explicitly that the database_to_transfer is green or give an error if it isn't blue or green.

Reply:
We do an explicit "is it other than 'green' or 'blue'" check above, so that is not possible